tests/Behat/Resources/suites.yml

```diff
@@ -0,0 +1,29 @@
+default:
+    suites:
+        reorders:
+            contexts_services:
+
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.product
+                - sylius.behat.context.setup.shipping
+                - sylius.behat.context.setup.payment
+                - sylius.behat.context.setup.order
+                - sylius.behat.context.setup.reorder
+                - sylius.behat.context.setup.shop_security
+                - sylius.behat.context.setup.promotion
+
+                - sylius.behat.context.transform.address
+                - sylius.behat.context.transform.channel
+                - sylius.behat.context.transform.lexical
+                - sylius.behat.context.transform.payment
+                - sylius.behat.context.transform.product
+                - sylius.behat.context.transform.promotion
+                - sylius.behat.context.transform.shared_storage
+                - sylius.behat.context.transform.shipping_method
+                - sylius.behat.context.transform.user
+
+                - sylius.behat.context.ui.shop.checkout.complete
+                - sylius.behat.context.hook.doctrine_orm
+                - sylius.behat.context.ui.shop.account
+        filters:
+            tags: "@managing_orders"
```

Comment:
Missing blank line at the end of file... and maybe we should provide out own "@reordering" tag?